### PR TITLE
create slideshow using dummy data 

### PR DESCRIPTION
**Pull request by Rahaf-96**
- install react-slideshow-image package, to create a slide component and give it a specific properties.
- create slideshow with dummy data, while every 3 projects displayed in one slide.
- styling the image of the project and the slide.

### CODE REVIEW

**Comment by Rahaf-96** [comment]
- create students slideshow contains studentcard for each student in the cohort.
- edit the spaces between slides.

**Comment by Rahaf-96** [comment]
remove dummy data from slideshow.

**Comment by someyaaltous** [comment]
Delete all the images @Rahaf-96

File: client/src/SharedComponent/Slideshow/Slideshow.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        }
+      </div>)
+    }
+    for (let i = 0; i < allStudents.length; i += 3) {
+      studentsSlide.push(<div className="one-slide" key={Date.now()}>
+        <div className='student'>{allStudents[i]}</div>
+        <div className='student'>{allStudents[i + 1]}</div>
+        <div className='student'>{allStudents[i + 2]}</div>
+      </div>)
+    }
```
Comment (someyaaltous) [inline client/src/SharedComponent/Slideshow/Slideshow.js:47]: Can you change the for loop to mapping

**Pull request merged**